### ISSUE TITLE
Add tbl_fst() function

### ISSUE THREAD

**Issue by krlmlr** [krlmlr/fstplyr#3]
that opens a table directly.

**Comment by MarcusKlik** [comment]
Hi @krlmlr, would you like this method to return a `tibble` (in memory) that contains the full table from the `fst` file (all columns and rows) ?

**Comment by krlmlr** [comment]
No, `tbl_fst("path/file.fst")` would just return `src_fst("path") %>% tbl("file")`.

**Comment by MarcusKlik** [comment]
I see, a shortcut to getting a table from the source, thanks!

**Comment by krlmlr** [comment]
We don't need to open metadata for all files here, but in principle the results are the same.
